### PR TITLE
setting the status to fix on the known issue

### PR DESCRIPTION
**Pull request by pmmalinov01**
Signed-off-by: Pavel Malinov <pmmalinov01@gmail.com>

#### What type of PR is this?

/kind cleanup
/kind documentation
#### Which issue(s) this PR fixes:

Fixes #
x-ref: #101728
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.21.md [hunk 1]
```diff
@@ -2111,4 +2111,4 @@ _Nothing has changed._
 
 ### Removed
 - rsc.io/quote/v3: v3.1.0
-- rsc.io/sampler: v1.3.0
\ No newline at end of file
+- rsc.io/sampler: v1.3.0
```
Comment (cici37) [inline CHANGELOG/CHANGELOG-1.21.md:2114]: Just curious what is the change here :)

Comment (pmmalinov01) [reply CHANGELOG/CHANGELOG-1.21.md:2114]: May new line? idk

**Pull request merged**